command-snapshot.json

```diff
@@ -239,6 +239,7 @@
     "flagChars": ["c", "d", "m", "n", "p", "t"],
     "flags": [
       "api-version",
+      "exclude-metadata",
```

Comment:
@shetzel , I think it should be `excluded-metadata`.  Because `exclude-metadata` sounds like a boolean, but this flag takes values.